### DISCUSSION TITLE
После апргрейта на про выдает ошибку при авторизации от одного из пользователей. Куда смотреть?

### DISCUSSION THREAD

**Discussion by maximav** [totumonline/totum-ru-issues-and-discussions#261] [Неопределенное]
(prep5) ОШИБКА: столбец "redirect_table_in_roles" не существует LINE 1: select redirect_table_in_roles->>'v' as redirect_table_in_ro... ^

**Comment by ttm-alexeypolunin** [comment]
Вы переключились с непоследней версии тотума. Обновите от пользователя `totum` из корневой папки установки `bin/totum git-update`

https://ru.docs.totum.online/update
